Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6
+FROM node:7
```

Comment:
misc: travis is building/testing with node 8, docker is set to 7.

Reply:
good catch, fixed!